ChangeLog

```diff
@@ -1,3 +1,58 @@
+3.18.2:
+	- 'N' or 'Ns' signal specs can now be used to sleep
+	  between signals sent by 'processes' promises
+	  (CFE-2207, ENT-5899)
+	- Added a test leveraging variablesmatching() to check that the agent
```

Comment:
why is this a changelog item? no behavior changed right?

Reply:
Right, dont need changelogs for tests my bad.